packages/airnode-protocol/contracts/rrp/authorizers/admin/AirnodeStablecoinPayment.sol

```diff
@@ -0,0 +1,289 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "../../../authorizers/interfaces/IRequesterAuthorizerWithManager.sol";
+import "./interfaces/IAirnodeFeeRegistry.sol";
+import "./interfaces/IERC20Extended.sol";
+import "./interfaces/IAirnodeStablecoinPayment.sol";
+
+/// @title The contract used to lock API3 Tokens in order to gain access to Airnodes
+contract AirnodeStablecoinPayment is IAirnodeStablecoinPayment {
+    string private constant ERROR_ZERO_CHAINID = "Zero chainId";
+    string private constant ERROR_ZERO_ADDRESS = "Zero address";
+    string private constant ERROR_NOT_AIRNODE = "Not airnode";
+    string private constant ERROR_ZERO_AMOUNT = "Zero amount";
+    string private constant ERROR_INSUFFICIENT_AMOUNT = "Insufficient amount";
+    string private constant ERROR_AIRNODE_NOT_OPTED_IN = "Airnode not opted in";
+
+    /// @notice Address of AirnodeFeeRegistry
+    address public airnodeFeeRegistry;
+
+    /// @notice The maximum whitelisting duration in seconds
+    uint64 public maximumWhitelistingDuration;
```

Comment:
I can't seem to find where this is being set 🤔

Reply:
yeah there is a setter function for this with its own access control role in the next commit